Improve the examples (#8)
cyberbotics/webots-cloud-simulation-examples: 4_custom_dockerfile/README.md

# 4 Custom Dockerfile

> This simulation is an example of how to load additional dependencies needed by the simulation.

Reviewer: ```suggestion
This simulation demonstrates how to load additional dependencies needed by the simulation.
```
